docs/install.rst

```diff
@@ -71,7 +71,7 @@ Homebrew/Linuxbrew installer takes care of pip for you.
 ===================
 
 Pipenv is a dependency manager for Python projects. If you're familiar
-with Node.js' `npm`_ or Ruby's `bundler`_, it is similar in spirit to those
+with Node's `npm`_ or Ruby's `bundler`_, it is similar in spirit to those
```

Comment:
Need to revert this line and add an exception

Reply:
yeah I suppose so :(